docker/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.1'
+
+services:
+  solr:
+    image: solr4
+    restart: always
+    ports:
+      - 8983:8983
+    volumes:
+      - data:/opt/solr/server/solr/mycores
+  db:
+    image: mysql:5
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+  web:
+    build: .
+    restart: always
+    ports:
+      - 8200:80
```

Comment:
why not "80:80"? changing ports usually makes it more confusing

Reply:
Because 80 is often taken on the host environment already - most devs here run a webserver on the host permanently. I'm just using the defaults from the CWP docker compose file here.